lib/board.rb

```diff
@@ -0,0 +1,57 @@
+class Board
+
+    attr_accessor :cells
+
+    def initialize
+      @cells = Array.new(9, " ")
```

Comment:
I would just call `reset!` here and delegate to that method to create the default empty board.